vivo-project/Vitro
# [VIVO-1893] i18n: Determinte and clean up all redundat files for new i18n support

**[JIRA Issue](https://jira.lyrasis.org/browse/VIVO-1893)**

**[2nd related PR](https://github.com/vivo-project/VIVO/pull/176)**

# What does this pull request do?
This PR removes all redudant files.
All languages files in VIVO- and Vitro-languages are available in the default installation of VIVO.
The default langauge is en-US. So there is no need for exra default files.


# How should this be tested?
* build a fresh vivo without languages to select
* there are some sites you can easily test
  * TermsOfUse (is it still available? is it in en-US?) 
  * Search-help (search with an empty field -> there should be a helptext)
  * MapOfScience 
  * AboutQRCodes (on a profilepage of a person, click the QRCode (and again) and then "What are QR-Codes")
* Account-Created-Mails
  * create a new user with an existing E-Mail, you should get an Mail with your account details
* the vivo_all.properties and all.properties were deleted 
  * check some examples or just click around in F12 modus (if something is missing, you get an error)
  * there properties are used almost everywhere

# Additional Notes:
* Now there is an empty folder ("Vitro/webapp/src/main/webapp/i18n/") -> should it be removed too?
